src/components/NcInputField/NcInputField.vue

```diff
@@ -53,7 +53,7 @@ For a list of all available props and attributes, please check the [HTMLInputEle
 						'input-field__input--error': error,
 						'input-field__input--pill': pill,
 					}]"
-				:value="value"
+				:value="value.toString()"
 				v-on="$listeners"
 				@input="handleInput">
```

Comment:
Just to raise awareness (as i spend quite a time to understand it :see_no_evil:):
- TextField don't listen for `@update:value`, but for `@input`
- As it wasn't modified, TextField always emits a string

Is it expected? As InputField is marked as internal component, we usually avoid to use it directly in the app
What should we do?
- update vue-lib docs, and 'allow' to use InputField?
- expand PR changes to TextField? (not sure about rest wrappers)